bitnami/memcached/Chart.yaml

```
@@ -16,4 +16,4 @@ name: memcached
 sources:
   - https://github.com/bitnami/bitnami-docker-memcached
   - http://memcached.org/
-version: 5.1.0
+version: 5.1.1
```

```suggestion
version: 5.2.0
```

As we are introducing a new parameter, this would be a minor.